web.Dockerfile

```diff
@@ -1,24 +1,20 @@
 # You can set the Swift version to what you need for your app. Versions can be found here: https://hub.docker.com/_/swift
-FROM swift:4.2 as builder
+FROM swift:5.0 as builder
 
 # For local build, add `--build-arg env=docker`
 # In your application, you can use `Environment.custom(name: "docker")` to check if you're in this env
 ARG env
 
-RUN apt-get -qq update && apt-get -q -y install \
-  tzdata \
-  && rm -r /var/lib/apt/lists/*
 WORKDIR /app
 COPY . .
-RUN mkdir -p /build/lib && cp -R /usr/lib/swift/linux/*.so /build/lib
+RUN mkdir -p /build/lib && cp -R /usr/lib/swift/linux/*.so* /build/lib
 RUN swift build -c release && mv `swift build -c release --show-bin-path` /build/bin
 
 # Production image
-FROM ubuntu:16.04
+FROM ubuntu:18.04
 ARG env
 RUN apt-get -qq update && apt-get install -y \
-  libicu55 libxml2 libbsd0 libcurl3 libatomic1 \
-  tzdata \
+  libatomic1 libicu60 libxml2 pkg-config git curl libz-dev \
```

Comment:
I don't see any reasons of adding Git in the default template, the idea is to try to keep it as small as possible.

Reply:
You're right, git was needed for me (but guess its just a dependency of it?) to work properly.
Will have a look if I can get this "smaller/more accurate".

Reply:
You can just remove it in this template, while adding it in your own :)

Reply:
No sorry I mean the vapor app didn't start without it, I guess it's missing another (system) dependency (which is installed through apt install git)

Reply:
but im checking that out right now, maybe I'm wrong

Reply:
Yea when removing git I'm hitting the error:
`./Run: error while loading shared libraries: libbsd.so.0: cannot open shared object file: No such file or directory`
added libbsd-dev and now it works again.